CONTRIBUTING.md

```diff
@@ -21,3 +21,56 @@ All submissions, including submissions by project members, require review. We
 use GitHub pull requests for this purpose. Consult
 [GitHub Help](https://help.github.com/articles/about-pull-requests/) for more
 information on using pull requests.
+
+## Getting your pull request through
+### Testing
+An essential part of getting your change through is to make sure all existing tests pass.
+
+#### Prerequisites
+* You will need maven installed to build and run tests.
+* You will need a GCP project to test on with the compute engine API enabled. Make sure you have all the relevant permissions for your project.
+* For development, we recommend IntelliJ.
+* **For Windows Images/VM's**, you will need Java and OpenSSH pre-installed. We have suggested startup-scripts for installing both if you do not want to pre-install, but OpenSSH installation tends to hang (this was tested with the Windows 2016 image provided by GCE).
```

Comment:
s/you will need Java and OpenSSH.../Java and OpenSSH...

Reply:
done